src/Sulu/Bundle/AdminBundle/Resources/js/components/Backdrop/README.md

```diff
@@ -18,6 +18,6 @@ intialState = {open: false};
 
 <div>
     <button onClick={() => setState({open: true})}>Open Backdrop</button>
-    <Backdrop isVisible={false} isOpen={state.open} onClick={() => setState({open: false})} />
+    <Backdrop isVisible={false} isOpen={!!state.open} onClick={() => setState({open: false})} />
```

Comment:
The make sense sometimes. Especially at the points where you really expect a boolean and not just a truthy type. At this point `undefined` and `false` make a difference.